sig-multicluster/hub-cluster-position-statement.md

```diff
@@ -0,0 +1,57 @@
+# Hub Cluster - SIG Multicluster Position Statement
+
+Author: Corentin Debains (**[@corentone](https://github.com/corentone)**), Google  
+Last Edit: 2025/01/25  
+Status: DRAFT
+
+## Goal
+To establish a standard definition for a central cluster that is leveraged by multicluster 
+controllers to manage multicluster applications or features across an inventory of clusters.
+
+## Context
+Multicluster controllers have always needed a place to run. This may happen in external
+proprietary control-planes but for more generic platforms, it has been natural for the
+Kubernetes community to leverage a Kubernetes Cluster and the existing api-machinery
+available. There has been a variety of examples of which we can quote ArgoCD, MultiKueue, Open Cluster Management
+or any of the Federation effort (Karmada, KubeAdmiral), all of them not-naming the "location"
+where they run or not aligning on the name (Admin cluster, Management Cluster, Command Cluster, Manager Cluster...). 
```

Comment:
Wow, hold on~
In fact, the SIG-Multicluster already have the concept and which was widely used in historical projects such as Federation and Kubefed. like:
https://github.com/kubernetes-retired/kubefed/blob/master/docs/concepts.md#kubefed-concepts

Reply:
thank you for bringing this up. Given kubefed has been retired, it may make sense to still have the definition published?
The nuance with the original kubefed "HostCluster" is that it's slightly more, not limited to where the kubefed API runs. Funnily, no one mentioned it so I wonder if it's really used today. I can mention it in the list of examples for non alignment?

Reply:
> Given kubefed has been retired, it may make sense to still have the definition published?

Kubefed was a key project prioritized by the SIG-Multicluster. Although the original contributors are no longer part of the SIG today, their efforts and contributions deserve to be fully acknowledged and shared.

Reply:
> Funnily, no one mentioned it, so I wonder if it's really used today.

I'm one of the maintainers of the Karmada project, which is developed in the continuation of Federation and Kubefed. Karmada inherited the basic concepts from them.  I can ensure that the concept is still used today.

Sorry for sharing the information so late, I just noticed this proposal this week.
I don't know when other guys started contributing to this group, I started with Kubefed. Although Kubefed did not succeed, but there were a lot of wealthy design docs left behind.

Reply:
> I can mention it in the list of examples for non alignment?

Actually, I think this proposal is discarding/changing legacy concepts, we need a good reason for it, because the legacy concept is still be used today.

Reply:
Hi @RainbowMango,
The original HostedCluster definition is very specific to the KubeFed project. As stated in the link you shared: "A cluster which is used to expose the KubeFed API and run the KubeFed control plane."

I'm not seeing how this PR redefines or overrides the prior HostedCluster definition. Would introducing a "Prior Art" section and referencing KubeFed's HostedCluster definition be acceptable?

Reply:
Hi Mike~, I'm not sure whether the definition of `Host Cluster` is specific to the KubeFed project, as Karmada inherited this definition from it :) and I'm trying to retrieve some design docs from SIG-MC(which is located in Google Docs), I remember there were a lot of discussions in the past. 

PS: I will be on public holiday from tomorrow and will get back next Tuesday.

Reply:
regarding a prior art section; Im adding Host Cluster in the context. Would that be enough or should we have a dedicated Prior Art section anyway? I was not thinking of trying to grasp the nuances of each term and each usage.

Reply:
I see, thanks @corentone. Looks great!

Given that renaming a concept is significant for a project that involves adjustments not only to the open-source project but also to user-facing products and documentation. 
I'd like to propose a discussion within the Karmada community, so, could we wait a bit longer to gather more feedback before making the final decisions? (Note that the voter participation in [the poll mentioned above](https://github.com/kubernetes/community/pull/8210#discussion_r1915759887) is very low comparing the huge number of contributors in multi-cluster areas. )

@jeremyot, @skitt , does it make sense to you?

Sorry again for missing the poll!